[1.12.2-Mohist]Can put Worms out on the ground in the server!

Mod version: 1.12.2-r151-2
Server Platform: Mohist Forge-1.12.2
Belows are error on the server terminal which occured when I right-clicked the grassblock, and I can't put Worms out.
`[19:12:52 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
        at net.minecraft.entity.Entity.func_174826_a(Entity.java:3170) ~[vg.class:?]
        at de.ellpeck.actuallyadditions.mod.entity.EntityWorm.<init>(EntityWorm.java:34) ~[EntityWorm.class:?]
        at de.ellpeck.actuallyadditions.mod.items.ItemWorm.func_180614_a(ItemWorm.java:66) ~[ItemWorm.class:?]
        at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:738) ~[ForgeHooks.class:0e603c8]
        at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:197) ~[aip.class:?]
        at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:612) ~[or.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:1138) ~[pa.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:61) ~[ma.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:13) ~[ma.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
        ... 5 more
[19:12:52 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
        at net.minecraft.entity.Entity.func_174826_a(Entity.java:3170) ~[vg.class:?]
        at de.ellpeck.actuallyadditions.mod.entity.EntityWorm.<init>(EntityWorm.java:34) ~[EntityWorm.class:?]
        at de.ellpeck.actuallyadditions.mod.items.ItemWorm.func_180614_a(ItemWorm.java:66) ~[ItemWorm.class:?]
        at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:738) ~[ForgeHooks.class:0e603c8]
        at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:197) ~[aip.class:?]
        at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:612) ~[or.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:1138) ~[pa.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:61) ~[ma.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:13) ~[ma.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
        ... 5 more
[19:12:52 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
        at net.minecraft.entity.Entity.func_174826_a(Entity.java:3170) ~[vg.class:?]
        at de.ellpeck.actuallyadditions.mod.entity.EntityWorm.<init>(EntityWorm.java:34) ~[EntityWorm.class:?]
        at de.ellpeck.actuallyadditions.mod.items.ItemWorm.func_180614_a(ItemWorm.java:66) ~[ItemWorm.class:?]
        at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:738) ~[ForgeHooks.class:0e603c8]
        at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:197) ~[aip.class:?]
        at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:612) ~[or.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:1138) ~[pa.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:61) ~[ma.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:13) ~[ma.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
        ... 5 more
[19:12:52 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
        at net.minecraft.entity.Entity.func_174826_a(Entity.java:3170) ~[vg.class:?]
        at de.ellpeck.actuallyadditions.mod.entity.EntityWorm.<init>(EntityWorm.java:34) ~[EntityWorm.class:?]
        at de.ellpeck.actuallyadditions.mod.items.ItemWorm.func_180614_a(ItemWorm.java:66) ~[ItemWorm.class:?]
        at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:738) ~[ForgeHooks.class:0e603c8]
        at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:197) ~[aip.class:?]
        at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:612) ~[or.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:1138) ~[pa.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:61) ~[ma.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:13) ~[ma.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
        ... 5 more
[19:12:52 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
        at net.minecraft.entity.Entity.func_174826_a(Entity.java:3170) ~[vg.class:?]
        at de.ellpeck.actuallyadditions.mod.entity.EntityWorm.<init>(EntityWorm.java:34) ~[EntityWorm.class:?]
        at de.ellpeck.actuallyadditions.mod.items.ItemWorm.func_180614_a(ItemWorm.java:66) ~[ItemWorm.class:?]
        at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:738) ~[ForgeHooks.class:0e603c8]
        at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:197) ~[aip.class:?]
        at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:612) ~[or.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:1138) ~[pa.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:61) ~[ma.class:?]
        at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(CPacketPlayerTryUseItemOnBlock.java:13) ~[ma.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
        ... 5 more
[19:12:52 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
[19:12:53 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
[19:12:53 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
[19:14:03 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
[19:14:03 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
[19:14:04 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NullPointerException
[19:14:04 FATAL]: Error executing task
 java.util.concurrent.ExecutionException: java.lang.NullPointerException
        at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_241]
        at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_241]
        at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:844) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:464) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:778) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:621) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]`

--- a/src/main/java/de/ellpeck/actuallyadditions/mod/entity/EntityWorm.java
+++ b/src/main/java/de/ellpeck/actuallyadditions/mod/entity/EntityWorm.java
@@ -21,6 +21,7 @@ import net.minecraft.block.IGrowable;
 import net.minecraft.block.state.IBlockState;
 import net.minecraft.entity.Entity;
 import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.math.AxisAlignedBB;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.World;
 import net.minecraftforge.common.IPlantable;
@@ -31,7 +32,7 @@ public class EntityWorm extends Entity {
 
     public EntityWorm(World world) {
         super(world);
-        this.setEntityBoundingBox(null);
+        this.setEntityBoundingBox(new AxisAlignedBB(0, 0, 0, 0, 0, 0));
     }
 
     public static boolean canWormify(World world, BlockPos pos, IBlockState state) {

